### PR TITLE
Fix minor typo

### PR DESCRIPTION
**Pull request by dubstack**
Fix for a really minor typo in development.md

@thockin PTAL.

### CODE REVIEW

**Comment by thockin** [comment]
It's not a typo.  You missed the '.' at the end of the git clone.  If you want this change (I made the same mistake the first time) you have to remove that . from the doc.

**Comment by dubstack** [comment]
Oh. I completely overlooked the dot. I missed it probably because markdown renders the '.' in blue. But yes this might happen with others aswell. WDYT? should I remove the dot aswell?

File: docs/devel/development.md [hunk 1]
```diff
@@ -230,9 +230,9 @@ separate dependency updates from other changes._
 
 ```sh
 export KPATH=$HOME/code/kubernetes
-mkdir -p $KPATH/src/k8s.io/kubernetes
-cd $KPATH/src/k8s.io/kubernetes
-git clone https://path/to/your/fork .
+mkdir -p $KPATH/src/k8s.io
+cd $KPATH/src/k8s.io
+git clone https://path/to/your/fork
```
Comment (thockin) [inline docs/devel/development.md]: Add a trailing line comment: `# assumes your fork is 'kubernetes'` then LGTM

Comment (dubstack) [inline docs/devel/development.md]: I also changed https://path/to/your/fork to https://path/to/your/kubernetes/fork to make it explicit.

**Comment by ixdy** [comment]
@k8s-bot test this please, issue #27462

**Comment by wojtek-t** [comment]
@k8s-bot test this please, issue #27462

**Comment by wojtek-t** [comment]
@k8s-bot e2e test this please, issue: #27462

**Pull request merged**